src/components/app-menu/AppMenu.tsx

```diff
@@ -1,38 +1,34 @@
-import { FC } from 'react'
+import Menu from '~/design-system/components/menu/Menu'
 
-import Menu, { MenuProps } from '@mui/material/Menu'
-
-import { spliceSx } from '~/utils/helper-functions'
-
-import { PositionEnum } from '~/types'
-import { styles } from '~/components/app-menu/AppMenu.styles'
-
-interface AppMenuProps extends MenuProps {
+interface AppMenuProps {
+  anchorEl: HTMLElement | null
+  menuList: {
+    title: string
+    onClick: () => void
+    graphics?: JSX.Element
+    sx?: object
+    isDisabled?: boolean
+  }[]
+  onClose: () => void
+  open?: boolean
+  sx?: object
   maxHeight?: number
-  menuList: JSX.Element | JSX.Element[]
 }
 
-const AppMenu: FC<AppMenuProps> = ({ maxHeight, menuList, sx, ...props }) => {
+const AppMenu = ({ anchorEl, menuList, onClose }: AppMenuProps) => {
```

Comment:
```suggestion
const AppMenu = ({ anchorEl, menuList, onClose, minWidth = 200, maxHeight = 400 }: AppMenuProps) => {
```

write it with  default value and than in `Menu` block pass this parameter like `maxHeight={maxHeight}`

Reply:
@Iarynovskyi done